### PR TITLE
css reset should not be inlined

### PR DESCRIPTION
**Pull request by lkrids**
The css reset is styling or resetting elements that are added by email clients.  These css rules shouldn't be inlined.

### CODE REVIEW

**Comment by smeric** [comment]
hi,

this pr is a bit old and i'm not sure it's still valuable... but you may find some references here :
http://templates.mailchimp.com/development/css/client-specific-styles/

thanks for your work.

File: email.html [hunk 1]
```diff
@@ -225,7 +225,7 @@
 		<a href="http://htmlemailboilerplate.com" target ="_blank" title="Styling Links" style="color: orange; text-decoration: none;">Coloring Links appropriately</a>
 
 		<!-- Gmail/Hotmail image display fix: Gmail and Hotmail unwantedly adds in an extra space below images when using non IE browsers.  This can be especially painful when you putting images on top of each other or putting back together an image you spliced for formatting reasons.  Either way, you can add the 'image_fix' class to remove that space below the image.  Make sure to set alignment (don't use float) on your images if you are placing them inline with text.-->
-		<img class="image_fix" src="full path to image" alt="Your alt text" title="Your title text" width="x" height="x" />
+		<img class="image_fix" src="full asdf to image" alt="Your alt text" title="Your title text" width="x" height="x" />
```
Comment (tenitski) [inline email.html:228]: Accidental edit here?

File: email.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- ***************************************************
 ********************************************************
 
-HOW TO USE: Use these code examples as a guideline for formatting your HTML email. You may want to create your own template based on these snippets or just pick and choose the ones that fix your specific rendering issue(s). There are two main areas in the template: 1. The header (head) area of the document. You will find global styles, where indicated, to move inline. 2. The body section contains more specific fixes and guidance to use where needed in your design.
+HOW TOasdf USE: Use these code examples as a guideliasdfne for formatting your HTML email. You may want to create your own template based on these snippets or just pick and choose the ones that fix your specific rendering issue(s). There are two main areas in the template: 1. The header (head) area of the document. You will find global styles, where indicated, to move inline. 2. The body section contains more specific fixes and guidance to use where needed in your design.
```
Comment (tenitski) [inline email.html:4]: Another accidental edit?

File: email.html [hunk 3]
```diff
@@ -206,7 +206,7 @@
 </head>
 <body>
 	<!-- Wrapper/Container Table: Use a wrapper table to control the width and the background color consistently of your email. Use this approach instead of setting attributes on the body tag. -->
-	<table cellpadding="0" cellspacing="0" border="0" id="backgroundTable">
+	<table cellpadding="0" cellspacing="0" border="5" id="backgroundTable">
```
Comment (tenitski) [inline email.html:209]: Why was it set to 5? There is no explanation anywhere...

**Pull request merged**